availability-store/src/lib.rs

```diff
@@ -42,6 +42,16 @@ pub struct Config {
 	pub path: PathBuf,
 }
 
+/// A record containing information about parachain extrinsic data to store.
```

Comment:
what is "parachain extrinsic data"? a `set_heads` call? a single extrinsic of a parachain? what about parachains that don't have single extrinsics?

Reply:
It's a piece of terminology we had since before Substrate, which essentially refers to the outgoing messages of the parachain, which has to remain available along with the block-data.

After Substrate introduced `Extrinsic`s, this naming became more confusing.

Reply:
Maybe rename to `IntermediateData` or even just `MessageData`?